src/app/directives/common/new-feature-indicator/new-feature-indicator-wrapper.component.html

```diff
@@ -0,0 +1,37 @@
+<ng-container *ngTemplateOutlet="template"></ng-container>
+<div class="tooltip">
+  <popper-content #popperTooltip class="fn-theme-tooltip">
+    <div class="tooltip-header">
+      <div class="title-wrapper">
+        <div>
+          <h5 class="tooltip-title">
+            {{ 'Help' | translate }}
+          </h5>
+        </div>
+      </div>
+      <div>
+        <ix-icon
+          class="close-icon"
+          name="mdi-close"
+          ixTest="tooltip-close"
+          (click)="popperTooltip.hide()"
```

Comment:
this is not good from accessibility point of view. let's wrap icon with material icon button.